Automattic/wp-calypso
# Hosting onboarding: Customise home experience for hosting devs

This PR provides basic boilerplate to tailor the home page view experience for developers. The views will change depending on what will be agreed suitable for a developer to see on the home page.

Related to https://github.com/Automattic/dotcom-forge/issues/2629



## Proposed Changes

* Create custom quick links component tailored to developers
* Remove daily blogging prompts (done in the backend).


## Testing Instructions

- Apply patch D112799-code 
- Verify daily blog post prompt is now shown
- Verify developer quick links are shown


![image](https://github.com/Automattic/wp-calypso/assets/47489215/e7b6f959-5a9b-4769-90fb-dbb30e3e3eae)



## Pre-merge Checklist
